app/src/pages/inside/projectSettingsPageContainer/content/notifications/messages.js

```diff
@@ -102,6 +102,10 @@ export const messages = defineMessages({
     id: 'AddNewCaseButton.addNewRuleButton',
     defaultMessage: 'Create Rule',
   },
+  notConfiguredTooltip: {
+    id: 'ButtonTooltip.notConfiguredTooltip',
```

Comment:
Let's rename it to something more specific. Because this tooltip message is only used in Notifications tab.